src/amo/api/abuse.js

```diff
@@ -23,7 +23,10 @@ export type AbuseReporter = {|
 export type ReportAddonParams = {|
   addonSlug: string,
   api: ApiState,
+  reporter_name: string | null,
+  reporter_email: string | null,
```

Comment:
nits: should be `reporterName` and `reporterEmail`.

Reply:
But those are the params we pass to the API function, so why can't we have them in camelcase?